### PR TITLE
Bulk delete

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

File: src/Bundle/test/src/Subscription/Entity/Subscription.php [hunk 1]
```diff
@@ -33,25 +33,25 @@
...
+        public ?int $id = null,
         #[Assert\NotBlank]
         #[Assert\Email]
         #[ORM\Column(name: 'name', nullable: false)]
         public ?string $email = null,
     ) {
     }
 
-    public function getId(): ?Uuid
+    public function getId(): ?int
```
Comment (loic425) [inline src/Bundle/test/src/Subscription/Entity/Subscription.php:63]: I've simplified this cause I had some troubles. But it works on a test project with an uuid on a mysql database.

Comment (lchrusciel) [reply src/Bundle/test/src/Subscription/Entity/Subscription.php:63]: What were the troubles?

File: src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php [hunk 2]
```diff
@@ -99,7 +99,7 @@ public function it_allows_updating_a_subscription(): void
             'sylius_resource[email]' => 'biff.tannen@bttf.com',
         ]);
 
-        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
+        //$this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
```
Comment (lchrusciel) [inline src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php]: Is it possible to international?

File: src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php [hunk 3]
```diff
@@ -137,6 +137,22 @@ public function it_allows_deleting_a_subscription(): void
...
 
+    /** @test */
+    public function it_allows_deleting_multiple_subscriptions(): void
+    {
+        $this->loadFixturesFromFile('subscriptions.yml');
+
+        $this->client->request('GET', '/admin/subscriptions');
+        $this->client->submitForm('Bulk delete');
+
+        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
```
Comment (lchrusciel) [inline src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php:148]: Is this test correct?

File: src/Bundle/test/src/Subscription/Entity/Subscription.php (hunk 1 above)
Comment (loic425) [reply src/Bundle/test/src/Subscription/Entity/Subscription.php:63]: The findById($ids) did not work with SQLite, but I have to check it that again later.
But if there is a bug with that on SQLite, we already have a bug on current implementation.

File: src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php (hunk 2 above)
Comment (loic425) [reply src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php]: That was a mistake, the redirection is OK.

File: src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php (hunk 3 above)
Comment (loic425) [reply src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php:148]: Yes, It checks that the redirect is ok. (it's a code from @Zales0123 on ScienceBookUiTest)

**Review by lchrusciel — APPROVED** [review]

**Pull request merged**